thirdparty/git.go

```diff
@@ -309,3 +309,11 @@ func FormGitUrl(host, owner, repo, token string) string {
 
 	return fmt.Sprintf("git@%s:%s/%s.git", host, owner, repo)
 }
+
+func FormGitUrlForApp(host, owner, repo, token string) string {
```

Comment:
nit: I know that the old function was already using `Url` but can we make it `URL` to be more consistent with our style guide?

Reply:
updated